client/src/index.css

```diff
@@ -73,7 +73,8 @@
 }
 
 .top-nav-container {
-  width: 800px;
```

Comment:
width of 800px is different from what we have for the body (width 70vw and max-width of 800px) so the nav bar would not move in sync w the body if the screen size changed and would sometimes be pushed off the page

Reply:
good catch!!